monitoring/tracing.mdx

```diff
@@ -40,8 +40,9 @@ For example, you are creating a AI travel agent to assist with booking trips. Th
 
 ### Instruction
 
-If you haven't authorize Baserun, please refer to the [getting started with SDK > Authentication](/uthentication) section for setup instructions.
+If you haven't authorize Baserun, please refer to the [getting started with SDK > Authentication](/getstartedwithSDK/authentication) section for setup instructions.
 
+{/* TODO: What does this refer to? The code doesn't show choosing any kind of environment */}
```

Comment:
We should check show does the SDK decide which environment it's running in, and put that into the documentation. I'm not 100% sure but it might look at the `ENV` environment variable for that.

Reply:
From what I can see in js and python sdks it looks like runs are marked to be either "TEST" or "PRODUCTION" but I have no idea what difference that makes. But that's not even something that you as a user have a direct control of. It kind of makes a guess if the code is run inside a test e.g. by pytest.

So...I still have no idea what this is about. Maybe it used to work differently and docs were not updated? No clue.